collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from collections import Counter
+from random import SystemRandom
+import requests
+from bases.FrameworkServices.SimpleService import SimpleService
+
+priority = 25
+update_every = 1
+
+DEFAULT_HOST = '127.0.0.1:19999'
+
+ORDER = [
+    'warning',
+    'critical'
+]
+
+CHARTS = {
+    'warning': {
+        'options': [None, 'Warning', 'warning', 'warning', 'alarms.warning', 'stacked'],
+        'lines': []
+    },
+    'critical': {
+        'options': [None, 'Critical', 'critical', 'critical', 'alarms.critical', 'stacked'],
+        'lines': []
+    }
+}
+
+
+class Service(SimpleService):
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+        self.random = SystemRandom()
+        self.host = self.configuration.get('host', DEFAULT_HOST)
+
+    @staticmethod
+    def check():
+        return True
+
+    def validate_charts(self, name, data, algorithm='absolute', multiplier=1, divisor=1):
+        for dim in data:
+            if dim not in self.charts[name]:
+                self.charts[name].add_dimension([dim, dim, algorithm, multiplier, divisor])
+
+    def get_data(self):
+
+        alarms = requests.get('http://{}/api/v1/alarms'.format(self.host)).json().get('alarms', {})
+        warning_data = dict(Counter([alarms[k]['name'] for k in alarms if alarms[k]['status'] == 'WARNING']))
+        critical_data = dict(Counter([alarms[k]['name'] for k in alarms if alarms[k]['status'] == 'CRITICAL']))
+        warning_data['_w'] = 0
+        critical_data['_c'] = 0
```

Comment:
what are these `_w`, `_c`?

Reply:
It seems i need to add a sort of dummy dimension at each timestep or else the charts either wont display initially, will fail to update and then go away. And/or if have periods of time with no alarms then i think without these dummy dimensions being updated to 0 each time the charts will not be shown as it would look like the collector is failing to collect any data.

Reply:
have just used one dim called `_` in most recent and added a note about it into the notes section of the README.md

Reply:
> will fail to update and then go away

it there is data later the chart is re-enabled

No need to add dummy dimensions, if you need count this should be a separate chart

```yaml
units: alarms
 dims:
   - warning
   - critical
```